.storybook/preview.js

```diff
@@ -5,8 +5,7 @@ import { vue3SourceDecorator } from "./decorators/vue3SourceDecorator";
 
 // Vue plugins
 import { createStore } from "vuex";
-import { createVueless } from "vueless";
-import NotifyServiceDefault from "vueless/ui.notify/services";
+import { createVueless } from "../src";
```

Comment:
`import { createVueless } from "vueless";` - keep as is  (there is an alias in vueless env)